--- mod_wsgi-3.4/configure.ac.apxsflags
+++ mod_wsgi-3.4/configure.ac
@@ -158,8 +158,11 @@ if test -x /usr/bin/lipo; then
     done
 fi
 
-CFLAGS="${CFLAGS1} ${CFLAGS2}"
-LDFLAGS="${LDFLAGS} ${LDFLAGS1} ${LDFLAGS2} ${LDFLAGS3}"
+CFLAGS3=`${APXS} -q CFLAGS`
+LDFLAGS4=`${APXS} -q LDFLAGS`
+
+CFLAGS="${CFLAGS1} ${CFLAGS2} ${CFLAGS3}"
+LDFLAGS="${LDFLAGS} ${LDFLAGS1} ${LDFLAGS2} ${LDFLAGS3} ${LDFLAGS4}"
 LDLIBS="${LDLIBS} ${LDLIBS1} ${LDLIBS2} ${LDLIBS3}"
 
 AC_SUBST(CFLAGS)
